fv_be/app/models/dictionary.py

```diff
@@ -0,0 +1,152 @@
+from django.db import models
+from django.utils.translation import gettext as _
+
+# FirstVoices
+from .base import BaseModel
+from .category import Category
+from .part_of_speech import PartOfSpeech
+
+
+class Note(BaseModel):
```

Comment:
Good point. If the model can be referred in other models, provided their notes field follow the same structure, to avoid duplication, we can move this out Dictionary file and make it a general table which can be referred to by any site. Or if the structure/field for different models is different but similar name, I agree with your point of renaming it to DictionaryNote.
Tagging @gmcauliffe @sarahfirstvoices @caforbes for their take on this.

Reply:
See below.